articles/frameworks/vue.md

```diff
@@ -1,3 +1,104 @@
 # Vue.js support
 
-TODO
+Visual Studio improved support for [Vue.js](https://vuejs.org/) framework, which enhances the experience when developing an application with Vue.js, JavaScript and TypeScript.
+
+## Feature overview
+
+In addition to the normal Visual Studio features when writing TypeScript or JavaScript, you'll see new behavior for Vue.js specific features:
+
+* Supported Script, Style and Template blocks in .vue files.
+* Recognize lang attribute on .vue files.
+
+## Create a Vue.js project with the vue-cli
+
+Vue.js provides and official CLI for quickly scaffolding projects. If you'll like to use it to create your application, follow the steps below to setup your developer environment.
```

Comment:
"__an__ official CLI", and "if __you would__ like to". (And "setup" should "set up").

Reply:
Changed.